src/open_inwoner/scss/components/Logo/Logo.scss

```diff
@@ -1,13 +1,9 @@
 .logo {
   &__image {
     max-width: 100%;
```

Comment:
This seems completely fine actually - I've done some really weird testing with strange SVG's and the only SVG's that do not behave well are the ones with inline-styling and **without** an explicit `viewbox` - so: if users upload correct and valid SVG's  then the styling behaves well (something to note for ourselves in case future users report problems with SVG).

Reply:
Ugh yes invalid or weird SVG's are a complication. The SVG used should at least look normal in a browser/viewer without margins (viewbox and transforms etc)